lib/classes/PluginManager.js

```diff
@@ -23,10 +23,22 @@ class PluginManager {
     this.loadServicePlugins(servicePlugins);
   }
 
+  validateCommands(commandsArray) {
+    // TODO: implement an option to get deeper than one level
+    if (!this.commands[commandsArray[0]]) {
+      const errorMessage =
+      `
```

Comment:
This newline can be removed once the automatic newline addition is done in the Error class.